### ISSUE TITLE
Load button for not-saved images missing

### ISSUE THREAD

**Issue by rugk** [threema-ch/threema-web#485]
## Steps to Reproduce (for bugs)

1. Someone sends a picture to you.
2. You cannot save it due to sd card problems or so.
3. Later, enter the sd card or so/fix whatever caused the problem, and 

## Current Behavior

Threema Web: Nothing is displayed. (= an empty message box instead of an image)
Threema: A "reload" button is displayed.

## Expected Behavior

The same reload button as in the app should be displayed and act the same.

## Your Environment

* Threema Web version: 1.8.2-gh

**Comment by lgrahl** [comment]
A note: Loading images can be made optional (for example on mobile connections), so this is not limited to *error on saving* conditions.

**Comment by rugk** [comment]
Yes, as it was described in way better terms and "descriptiveness" in issue https://github.com/threema-ch/threema-web/issues/523. :wink:

**Comment by lgrahl** [comment]
I believe this requires an app update. I tried resolving it but couldn't download the blob.

**Comment by lgrahl** [comment]
Correct me if I'm wrong but I don't think iOS even has this setting.

**Comment by lgrahl** [comment]
Filed an internal bug: ANDR-955
